core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/workspacemanager/WorkspaceManagerDAO.scala

```diff
@@ -22,8 +22,6 @@ trait WorkspaceManagerDAO {
                                       ctx: RawlsRequestContext
   ): CreatedWorkspace
 
-
-
```

Comment:
scalafmt is failing on this PR, after I updated to `develop`. It looks like these extra lines were added in #2552, but scalafmt passed on that PR. I don't know why it passed there but is failing here, but I'll remove the extra lines.